io.openems.edge.common/src/io/openems/edge/common/currency/Currency.java

```diff
@@ -0,0 +1,35 @@
+package io.openems.edge.common.currency;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum Currency implements OptionsEnum {
+	UNDEFINED(-1, "-"), //
+	EUR(0, "€"), //
+	SEK(1, "kr"), //
+	USD(2, "$");
```

Comment:
Is there concrete use for USD already? Otherwise we could just leave it out for now, as we would have to cover it everywhere.